### PR TITLE
fix(sig-ci): Updating sig day

### PR DESCRIPTION
**Pull request by gal-yardeni**
Rescheduling the sig meetings to Wednesdays.

### CODE REVIEW

File: sig-ci/README.md [hunk 1]
```diff
@@ -8,7 +8,8 @@ This SIG is focused on CI (continuous integration) experience within Spinnaker.
 ## Meetings
 
 * Regular SIG Meeting
-  * Every other Tuesday, 1 PM EST / 10AM PST
+
```
Comment (dbyron0) [inline sig-ci/README.md]: Did the Tuesday meeting ever make it to the [spinnaker google calendar](https://calendar.google.com/calendar/embed?src=spinnaker.io_tr65rjf5mfij7p6vucprkhulcc%40group.calendar.google.com&ctz=America%2FLos_Angeles)?  Trying to find out if I've missed any meetings.

Comment (gal-yardeni) [reply sig-ci/README.md]: Yes - the same Tuesdays as the spinnaker as code sig (but morning).

Comment (dbyron0) [reply sig-ci/README.md]: OK, I think I figured it out...I wasn't looking far enough into the future.  The first one I see is currently Wednesday Feb 12th at 10AM PST.

Comment (dbyron0) [reply sig-ci/README.md]: BTW, noticed https://github.com/spinnaker/governance/pull/96 along the way.

**Pull request merged**